internal/k8s/global_configuration.go

```diff
@@ -74,11 +74,11 @@ func (lbc *LoadBalancerController) syncGlobalConfiguration(task task) {
 	var validationErr error
 
 	if !gcExists {
-		glog.V(2).Infof("Deleting GlobalConfiguration: %v\n", key)
+		nl.Debugf(lbc.logger, "Deleting GlobalConfiguration: %v\n", key)
```

Comment:
Should this be InfoF then? as its v2 previously

Reply:
according to the levels package 
```
	// LevelDebug - Debug Level Logging same as glog.V(2)
	LevelDebug = slog.LevelDebug
```

Reply:
So what does this mean for the V(3) ones?
Earlier this pr this
`glog.V(3).Infof("Adding GlobalConfiguration: %v", gc.Name)` 
changed to 
`nl.Debugf(lbc.logger, "Adding GlobalConfiguration: %v", gc.Name)`

And here a v2 Info
`glog.V(2).Infof("Deleting GlobalConfiguration: %v\n", key)`
is changing also to Debugf 
`nl.Debugf(lbc.logger, "Deleting GlobalConfiguration: %v\n", key)`

Is this intentional?

Reply:
yep, there isn't a level between Debug and Info in logging libraries.  The alternative would be to have `glog.V(3)` as TRACE, and `glog.V(2)` as DEBUG